lib/omnibus/licensing.rb

```diff
@@ -25,6 +27,7 @@ class Licensing
     include Sugarable
 
     OUTPUT_DIRECTORY = "LICENSES".freeze
+    CACHE_DIRECTORY = "license-cache".freeze
```

Comment:
Might want to put the word "license" or "licensing" in here and the related methods. I thought this was a different cache when I was reading the code.